src/en/send.md

```diff
@@ -353,3 +341,145 @@ If the request is not successful, the response body is `json`, refer to the tabl
 |`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM/TEST/LIVE of 1000 requests per 60 seconds"`<br>`}]`|Refer to [API rate limits](limits.md) for more information|
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](limits.md) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|GC Notify was unable to process the request, resend your notification.|
+
+
+## Sending notifications in bulk
+
+```
+POST /v2/notifications/bulk
+```
+
+Send a batch of notifications, up to 50,000 recipients at a time, for a single template. You can schedule to send notifications up to 4 days in the future.
+
+### Request body
+
+```json
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "rows": [
+    ["email address", "name"],
+    ["alice@example.com", "Alice"],
+    ["bob@example.com", "Bob"]
+  ],
+  "scheduled_for": "2021-06-08T15:15:00", # optional string
+  "sender_id": "f025b1a9-63af-43e8-b969-627bfe544bba" # optional string
+}
+```
+
+### Arguments
+
+**name (required)**
+
+The `name` of your bulk sending. Used to identify this batch of notifications later on.
+
+<Content :page-key="$site.pages.find(p => p.relativePath === 'en/_arg_template_id.md').key"/>
+
+**rows (required)**
+
+An array of arrays. The first line is the header and should include at least `email address` if you're sending an email template or `phone number` if you're sending a text message template. The other column headers should match placeholder fields of your template.
```

Comment:
What does "an array of arrays" mean?

Reply:
Well, it's an array, containing arrays. Like
```
[
  [1, 2, 3],
  [4, 5]
]
```